src/include/main/db_main.h

```diff
@@ -367,8 +373,9 @@ class DBMain {
         log_manager->Start();
       }
 
-      auto txn_layer = std::make_unique<TransactionLayer>(common::ManagedPointer(buffer_segment_pool), use_gc_,
-                                                          common::ManagedPointer(log_manager));
+      auto txn_layer =
+          std::make_unique<TransactionLayer>(common::ManagedPointer(buffer_segment_pool), use_gc_,
```

Comment:
I'm completely unsure why this might not be covered by tests but every line in this function is... thoughts?

Reply:
Also, codecov in general seems to be confused since it only says ~68% of your diff is hit